recap/converters/json_schema.py

```diff
@@ -44,6 +45,9 @@ def to_recap(
         alias_strategy = alias_strategy or JSONSchemaConverter.urn_to_alias
         json_schema = json.loads(json_schema_str)
         recap_schema = self._parse(json_schema, alias_strategy)
+        if isinstance(recap_schema, UnknownType):
```

Comment:
This feels hacky. I think we should probably refactor json_schema to use a similar strategy to the avro converter; namely, creating a _parse that returns `RecapType` and using that in to_recap.